management/server/sqlite_store.go

```diff
@@ -483,12 +485,57 @@ func (s *SqliteStore) SaveUserLastLogin(accountID, userID string, lastLogin time
 	return s.db.Save(user).Error
 }
 
-// Close is noop in Sqlite
+// Close closes the underlying DB connection
 func (s *SqliteStore) Close() error {
-	return nil
+	db, err := s.db.DB()
+	if err != nil {
+		return fmt.Errorf("get db: %w", err)
+	}
+	return db.Close()
 }
 
 // GetStoreEngine returns SqliteStoreEngine
 func (s *SqliteStore) GetStoreEngine() StoreEngine {
 	return SqliteStoreEngine
 }
+
+// CalculateUsageStats returns the usage stats for an account
+// start and end are inclusive.
+func (s *SqliteStore) CalculateUsageStats(ctx context.Context, accountID string, start time.Time, end time.Time) (*AccountUsageStats, error) {
+	stats := &AccountUsageStats{}
+
```

Comment:
It might make sense to wrap everything to `s.db.Transaction`

Reply:
Addressed in 9ea26a1f4198620e5d186cf897e534dc7aa93f84